src/EventJournal.md

```diff
@@ -0,0 +1,75 @@
+## Event journal
+
+The event journal is a distributed data structure that stores the history of mutation actions on data structures such as map or cache. Each action on the map or cache which modifies its contents (such as `put`, `remove` or scheduled tasks which are not triggered by using the public API) will create an event which will be stored in the event journal. The event will store the event type as well as the key, old value and updated value for the entry (when applicable). As a user, you can only append to the journal indirectly by using the map and cache methods or by configuring expiration and eviction. 
```

Comment:
I can see we've removed some text about evictions/expirations causing different entries in the event journal for the same entry but different replicas.

As the Event Journal is a Partitioned Ringbuffer presumably the replica entry lives in one place and the primary entry lives in another part of the ringbuffer.

Are we saying that the two entries are "eventually" consistent?